### PR TITLE
[forge-viewer] added FragmentList, GeometryList

### PR DESCRIPTION
**Pull request by liskaj**
- added FragmentList class
- added GeometryList class
- updated InstanceTree class

### CODE REVIEW

File: types/forge-viewer/index.d.ts [hunk 1]
```diff
@@ -1115,6 +1116,50 @@ declare namespace Autodesk {
             function getHtmlTemplate(url: string, callback: (error: string, content: string) => void): void;
             function lerp(x: number, y: number, t: number): number;
 
+            class FragmentList {
```
Comment (rbuckton) [inline types/forge-viewer/index.d.ts:1119]: Is `FragmentList` a value exported by the namespace `Private`, or should this be declared as an `interface`? The project link at the top of this file points to a page that no longer exists and the only reference I can find is this: https://forge.autodesk.com/en/docs/viewer/v7/reference/Private/ which does not include `FragmentList`.

File: types/forge-viewer/index.d.ts [hunk 2]
```diff
@@ -1115,6 +1116,50 @@ declare namespace Autodesk {
...
+              originalColors: any[];
+              pointsHidden: boolean;
+              themingOrGhostingNeedsUpdate: any[];
+              transforms: any;
+              useThreeMesh: boolean;
+              vizflags: Uint32Array;
+              vizmeshes: THREE.Mesh[];
+            }
+
+            class GeometryList {
```
Comment (rbuckton) [inline types/forge-viewer/index.d.ts:1149]: Similar question as with `FragmentList`, above. Is `GeometryList` a value exported by the namespace `Private`, or should this be declared as an `interface`?

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/forge-viewer/index.d.ts (hunk 1 above)
Comment (liskaj) [reply types/forge-viewer/index.d.ts:1119]: Good point. Will fix this.

File: types/forge-viewer/index.d.ts (hunk 2 above)
Comment (liskaj) [reply types/forge-viewer/index.d.ts:1149]: Same as above.

**Review by rbuckton — APPROVED** [review]

**Comment by liskaj** [comment]
Ready to merge

**Pull request merged**